tests/browser_harness.html

```diff
@@ -23,17 +23,24 @@
     }
   </style>
   <script>
+    var COMMAND_PREFIX = 'COMMAND:';
     var counter = 0;
     function check() {
       var request = new XMLHttpRequest();
       request.open('GET', '/check');
       request.send(null);
       request.addEventListener("load", function() {
-        if (request.responseText != 'False') {
-          iframe.src = request.responseText;
+        if (request.responseText.startsWith(COMMAND_PREFIX)) {
+          var url = request.responseText.substr(COMMAND_PREFIX.length);
+          console.log('commanded to load ' + url);
```

Comment:
Is this log intended to be permanent, or was this used temporarily for debugging? (good either way)

Reply:
Hmm, yeah, kind of both really. I can remove later if it's annoying.

Reply:
Looking at https://circleci.com/gh/emscripten-core/emscripten/91721?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link this does have the effect of over-verbosing the print, so CI says

```
ok
test_sdl_canvas_proxy (test_browser.browser) ... [browser launch: test.html ]
[queue command: http://localhost:8888/test.html /tmp/emscripten_test_browser_kzxUvb ]
[simple HTTP serving: /test.html ]
[simple HTTP serving: /test.js ]
[simple HTTP serving: /test.data ]
[simple HTTP serving: /test.wasm ]
[simple HTTP serving: /sdl_canvas_proxYour output is too large to display in the browser. Only the first 400000 characters are displayed.
Download the full output as a file.
```

perhaps good to drop the log it looks like.

Reply:
Thanks, yeah, that is annoying. Fix in #8062